packages/components/src/components/domainproperties/DomainForm.tsx

```diff
@@ -86,6 +87,8 @@ interface IDomainFormInput {
     domainIndex?: number
     successBsStyle?: string
     setFileImportData?: (file: File) => any // having this prop set is also an indicator that you want to show the file preview grid with the import data option
+    hideAddFieldsButton?: boolean
```

Comment:
Any reason not to have the hideAddFieldsButton property be part of the domainFormDisplayOptions?